calc/states_for_test.go

```diff
@@ -1152,6 +1152,43 @@ var vxlanTunnelIPDelete = vxlanWithBlock.withKVUpdates(
 	},
 ).withVTEPs()
 
+// Two nodes with VXLAN and CrossSubnet.
+var vxlanWithCrossSubnet = empty.withKVUpdates(
+	KVPair{Key: ipPoolKey, Value: &ipPoolWithVXLANCrossSubnet},
+	KVPair{Key: localIPAMBlockKey, Value: &localIPAMBlock},
+	KVPair{Key: localHostIPKey, Value: &localHostIP},
```

Comment:
I think we want two states in order to simulate the original issue:

State 1:
 - Remote node IP and tunnel address
 - Remote node IPAM block
 - IP pool

State 2 (in addition to state 1):
- Local node IP and tunnel address

The way it's formulated right now, the test sends all of these in as a single state transition (vxlanWithCrossSubnet includes all of the objects in both state1 and state1), and so we can't guarantee the ordering in the original issue.

Reply:
Ok, updated.